src/msw/graphicsd2d.cpp

```diff
@@ -2947,17 +2947,17 @@ class wxD2DRenderTargetResourceHolder : public wxD2DResourceHolder<ID2D1RenderTa
     // The default implementation works for all render targets, but the D2D 1.0
     // render target holders shouldn't need to override it, since none of the
     // 1.0 render targets offer a better version of this method.
-    virtual void DrawBitmap(ID2D1Bitmap* bitmap, D2D1_POINT_2F offset,
-        D2D1_RECT_F imageRectangle, wxInterpolationQuality interpolationQuality,
+    virtual void DrawBitmap(ID2D1Bitmap* bitmap,
+        D2D1_RECT_F srcRect, D2D1_RECT_F destRect,
```

Comment:
It looks like it would be better to pass those by const reference, i.e.
```suggestion
        const D2D1_RECT_F& srcRect, const D2D1_RECT_F& destRect,
```